synfig-core/src/modules/lyr_freetype/lyr_freetype.h

```diff
@@ -88,7 +88,7 @@ class Layer_Freetype : public synfig::Layer_Composite, public synfig::Layer_NoDe
 
 	synfig::Color color_func(const synfig::Point &x, int quality=10, synfig::ColorReal supersample=0)const;
 
-	mutable std::mutex mutex;
+	std::mutex mutex;
```

Comment:
Why did you remove `mutable`? In C++11, it's totally OK to use mutable with thread-safe variables.

Reply:
http://herbsutter.com/2013/01/01/video-you-dont-know-const-and-mutable/

https://stackoverflow.com/questions/4127333/should-mutexes-be-mutable

That said, I think we should audit all `mutable` and `const_cast` on code to fix them ;)